### PR TITLE
fix: standardize user-agent header

### LINKED ISSUE #1223: Standardize user-agent header
Format should be: `elasticapm-$language/$agent_version`

see: https://github.com/elastic/apm/issues/88

### PR DESCRIPTION
**Pull request by yuxizhe**
close #1223
### Checklist

- [x] Implement code
- [ ] Add tests
- [ ] Update documentation

### CODE REVIEW

**Comment by nik9000** [comment]
@elasticmachine, run elasticsearch-ci/docs

**Comment by nik9000** [comment]
@elasticmachine, run elasticsearch-ci/docs-build if you can. But i have no idea where that is configured.

**Comment by nik9000** [comment]
> @elasticmachine, run elasticsearch-ci/docs-build if you can. But i have no idea where that is configured.

`elasticsearch-ci/docs-build` was configured in the branch protection settings. I've switched it to `elasticsearch-ci/docs` because that is the new name.

File: lib/config.js [hunk 1]
```diff
@@ -10,7 +10,12 @@ var truncate = require('unicode-byte-truncate')
 var entries = require('object.entries') // TODO: Remove when Node.js 6 is no longer supported
 
 var version = require('../package').version
-var userAgent = 'elastic-apm-node/' + version
+var packageName = require('../package').name
+// Standardize user-agent header. Only use "elasticapm-node" if it matches "elastic-apm-node". 
+if (packageName === 'elastic-apm-node') {
+  packageName = 'elasticapm-node'
+}
+var userAgent = `${packageName}/${version}`
```
Comment (Qard) [inline lib/config.js]: @axw Do you think we should be doing this package name check or should we just _always_ call it `elasticapm-node`? I'm not sure there's a significant value to being able to differentiate forks. 🤔

Comment (axw) [reply lib/config.js]: I'm not personally fussed either way. Could be useful for debugging? OTOH, we can probably tell from stack traces anyway right? Are there any downsides?

Comment (Qard) [reply lib/config.js]: Not really, just extra code. 🤷‍♂

**Comment by Qard** [comment]
jenkins run the tests please

File: lib/config.js (hunk 1 above)
Comment (axw) [reply lib/config.js]: Let's leave it in, could be useful for diagnosing issues in the future.

**Comment by Qard** [comment]
jenkins run the tests

**Comment by watson** [comment]
Isn't this a breaking change?

**Comment by Qard** [comment]
In what way? It was considered _not_ breaking in prior discussion. 🤔

**Comment by watson** [comment]
Ok. I tried to look for a discussion around it in the issues but didn't find anything?
I was just wondering if anybody could be depending on the user-agent.

**Pull request merged**